### ISSUE TITLE
Add default minio setup if user disabled aws s3 setup

### ISSUE THREAD

**Issue by okamara** [zebrunner/zebrunner#397]
Idea: Non technical User that don't have aws s3 may be don't know what minio storage means. We need to make minio set up by default if user answer n for aws s3(1st question) 
Add message to nitify user that minio will be set up for your now by default.
Case:
During 1st set up add aws s3 - y/n, 
if user select n,
then  minio will be set up with notification message for user

**Comment by vdelendik** [comment]
fixed.
among the changes:
1. asking about using aws s3 bucket (by default "n" i.e. disabled)
```
Use AWS S3 bucket for storing test artifacts (logs, video, screenshots etc)? Embedded Minio Storage can be configured if you don't have Amazon account.
Use? y/n [n]:
```
-> if user say "n" we setup automatically minio storage, if "y" - we ask aws s3 bucket details

2. asking about SMTP integration (disabled by default):
```
Use SMTP for emailing test results?
Use? y/n [n]:
```
-> only after answerring "y" we ask about host, port, user creds etc

3. asking about GITHUB integration (disabled by default):
```
Use GitHub integration for Test Launchers in Reporting UI?
Use? y/n [n]:
```
-> only after answering "y" we ask about host, client id and secret


in scope of testing such scenarios should be tested:
1. setup using default arguments
-> reporting is available and configured using minio, no smtp and github integrations available

2. setup using aws s3 bucket
-> reporting is available, artifacts are stored in aws s3 bucket, no smtp,  no github integrations available 

3. setup using valid smtp integration
-> email notifications should be sent after test runs

4. setup using valid github integrations
-> Test Launcher "add repo" functionality should wokrk as expected

**Comment by okamara** [comment]
Verified. Done. Created a separate ticket for missing Zafira report in aws s3 setup.
